File: team_origin/Entities/Notifications/Notification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace team_origin.Entities.Notifications
{
    public class Notification
    {
        public int NotificationId { get; set; }
        public int NotificationTypeId { get; set; }
        public string NotificationDetails { get; set; }
        public bool NotificationAcknowledged  { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDateTime { get; set; }
        public User User { get; set; }
        public NotificationType NotificationType { get; set; }
        public UserNotificationRef UserNotificationRef { get; set; }
    }
}

File: FriendsTracker/app/shared/user/notification.ts
export class Notification {
    public NotificationId: number;  
    public NotificationTypeId: number;
    public NotificationDetails: string;
    public NotificationAcknowledged: boolean;   
    public CreatedBy: string;
    public CreatedDateTime: Date; 
}
export class FriendRequestResponse{
    public notification: Notification;
    public LoggedInUserId: string
}